### PR TITLE
Reword paragraph about SCF's GitHub user

### PR DESCRIPTION
**Pull request by r-gaia-cs**

### CODE REVIEW

File: 04-open.md [hunk 1]
```diff
@@ -199,7 +199,10 @@ than to run many smaller services to the same standard.
 It's also easier for people to collaborate:
 using a popular service can help connect your project with communities already using the same service.
 
-As an example, the [Software Carpentry organization](https://github.com/swcarpentry/) on GitHub includes the [source code for this page](https://github.com/swcarpentry/git-novice/blob/gh-pages/04-open.md). Anyone with a GitHub account can suggest changes to this text.
+As an example,
+Software Carpentry Foundation [has a user on GitHub](https://github.com/swcarpentry/)
```
Comment (jiffyclub) [inline 04-open.md]: https://github.com/swcarpentry/ is an organization, not a user. I don't know if that distinction is apparent to readers at this point.

We could gloss over that by rephrasing this as "As an example, Software Carpentry [is on GitHub](https://github.com/swcarpentry/), where you can find the [source for this page](https://github.com/swcarpentry/git-novice/blob/gh-pages/04-open.md)."

Comment (r-gaia-cs) [inline 04-open.md]: > https://github.com/swcarpentry/ is an organization, not a user.

I know that. I want to simplify for novice users.

> We could gloss over that by rephrasing this (...)

Done.

File: 04-open.md [hunk 2]
```diff
@@ -199,7 +199,10 @@ than to run many smaller services to the same standard.
 It's also easier for people to collaborate:
 using a popular service can help connect your project with communities already using the same service.
 
-As an example, the [Software Carpentry organization](https://github.com/swcarpentry/) on GitHub includes the [source code for this page](https://github.com/swcarpentry/git-novice/blob/gh-pages/04-open.md). Anyone with a GitHub account can suggest changes to this text.
+As an example,
+Software Carpentry Foundation [is on GitHub](https://github.com/swcarpentry/)
```
Comment (jiffyclub) [inline 04-open.md]: I think this should be either "the Software Carpentry Foundation" or plain "Software Carpentry".

Comment (r-gaia-cs) [inline 04-open.md]: > I think this should be either "the Software Carpentry Foundation" or plain "Software Carpentry".

Changed to plain "Software Carpentry".

**Pull request merged**